plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml

```diff
@@ -89,29 +89,58 @@ steps:
     parallel: false
 
 
+  - name: configure a simulated fabric
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
+    run: | 
+      cd /auto/UFM/tmp/ufm_verification/tests/test_with_sim/performance
+      IP_SERVER=$(ping $SERVER_HOST -c 1 -s 16 | grep -oE '[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+' | head -1)
+      python run_simulator.py -s $IP_SERVER --topology_path ibnetdiscover_director.txt
+    parallel: false
+
+
   - name: run client tests
     agentSelector: "{nodeLabel: 'UFM-POC'}"
     run: |
       service rsyslog restart
       sleep 30
       export SERVER_HOST=$SERVER_HOST
       cd plugins/UFM_NDT_Plugin/tests
-      python3 ndt_plugin_test.py --host $SERVER_HOST || true
+      python3 ndt_plugin_test.py --host $SERVER_HOST
     parallel: false
 
 
   - name: get ndt logs
     agentSelector: "{nodeLabel: 'UFM-POC'}"
     run: |
       mkdir $WORKSPACE/logs/
-      plugins/UFM_NDT_Plugin/.ci/do_get_logs.sh || true
+      plugins/UFM_NDT_Plugin/.ci/do_get_logs.sh
     archiveArtifacts: 'logs/ndt.log'
     parallel: false
 
 
   - name: remove plugin end
     agentSelector: "{nodeLabel: 'UFM-POC'}"
-    run: plugins/UFM_NDT_Plugin/.ci/do_remove_plugin.sh || true
+    run: plugins/UFM_NDT_Plugin/.ci/do_remove_plugin.sh
+    parallel: false
+
+  - name: back to real fabric
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
+    run: | 
+      plugins/UFM_NDT_Plugin/.ci/real_fabric.sh
+      opensm_pid=$(pgrep -x opensm)
+      kill -9 $opensm_pid
+      ps -ef | grep -i sm
+    parallel: false
+
+  - name: draft
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
+    run: |
+      plugins/UFM_NDT_Plugin/.ci/draft.sh
+      /etc/init.d/ufmd stop
+      ps -ef | grep -i sm
+      opensm_id=$(cat /var/run/opensm_sim.pid)
+      kill -9 $opensm_id
+      plugins/UFM_NDT_Plugin/.ci/draft.sh
     parallel: false
 
 pipeline_start:
```

Comment:
please don't remove the build in pipeline_stop, these build artifacts could be useful for debugging